prow/getting_started_deploy.md

```diff
@@ -2,13 +2,42 @@
 
 This document will walk you through deploying your own Prow instance to a new Kubernetes cluster. If you encounter difficulties, please open an issue so that we can make this process easier.
 
-## How to turn up a new cluster
+Prow runs in any kubernetes cluster. Our `tackle` utility helps deploy it correctly, or you can perform each of the steps manually.
 
-Prow should run anywhere that Kubernetes runs. Here are the steps required to
-set up a basic prow cluster on [GKE](https://cloud.google.com/container-engine/).
-Prow will work on any Kubernetes cluster, so feel free to turn up a cluster
-some other way and skip the first step. You can set up a project on GCP using
-the [cloud console](https://console.cloud.google.com/).
+Both of these are focused on [Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) but should work on any kubernetes distro with no/minimal changes.
+
+# Tackle deployment
+
+Prow's tackle utility walks you through deploying a new instance of prow in a couple minutes, try it out!
+
+You need three things:
+1) The prow tackle command.
+2) a [personal access token][1] for the github bot user you want to act as prow
+  - Prow uploads this token into a secret in your cluster
+3) Optionally, credentials to a kubernetes cluster
+  - Otherwise the tackle command can help you create one
+
+Then run the following and follow on-screen instructions:
+
+```bash
+# go get -u k8s.io/test-infra/prow/cmd/tackle && tackle
```

Comment:
I'm not sure what you're trying to suggest they do with this line (especially the `&& tackle` part).

Reply:
I want to give bazel instructions but technically this tool does not require bazel, so want to provide that fallback.

Is there a way to combine go get -u && go run? How would you like me to write this?

Reply:
I think the command as-written is fine in itself, but giving two ways to achieve the same goal without explaining that's what you're doing is confusing. I'd like to see a note explaining why they're both there (maybe just swapping the order plus a "or without bazel:" prefix or similar).

In other places where you have added commented-out commands (e.g. `# openssl rand -hex 20 > /path/to/hook/secret`), it's a suggestion of something you probably will need to do, rather than an alternate approach.

The same applies to the `add-hook` example further down.